### ISSUE TITLE
Run failed: deploy-book

### ISSUE THREAD

**Issue by rvosshall** [Probabilistic-ML/lecture-notes#13]
Previous requirements.txt results in the following error:

```
Exception occurred:
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/pydata_sphinx_theme/bootstrap_html_translator.py", line 34, in visit_table
    self._table_row_index = 0
AttributeError: can't set attribute
```

See also [Incompatibility with sphinx 4.3.0](https://github.com/executablebooks/sphinx-book-theme/issues/427).

Issue seems to be fixed in [release 0.7.2 of pydata-sphinx](https://github.com/pydata/pydata-sphinx-theme/commit/ae888d1ce2b8b7fd0016d1a0536defb5eff2b47c), but installation of jupyterpy-book uses data-sphinx-theme-0.6.3.

Use of sphinx 4.2 yields intermediate solution.

**Comment by canbooo** [comment]
> Use of sphinx 4.2 yields intermediate solution.
What does this mean. Could we solve this by changing our workflow script?

**Comment by canbooo** [comment]
Commit cfae9ee fixes this for now. Reopen if the incompatibility is fixed and you feel like we should use the most recent version.

**Comment by kousu** [comment]
btw as soon as the next `sphinx-book-theme` is released, it will use `pydata-sphinx-theme-0.7.2`, and that will fix this bug. And a release should be coming in a few days it sounds like!

https://github.com/executablebooks/sphinx-book-theme/pull/429#pullrequestreview-804056728
